server/controllers/comment.controller.js

```diff
@@ -70,6 +67,41 @@ class CommentController {
       return Response(res, 201, 'thread comment added', response);
     }
   }
+
+  /**
+   * Like comment controller
+   * @param {Object} req
+   * @param {Object} res
+   * @returns {Object} response
+   */
+  static async likeComment(req, res) {
+    const { id } = req.user;
+    const { commentid } = req.params;
+
+    const commentResponse = await comments.findOne({
+      attributes: ['comment'],
+      where: { id: commentid },
+    });
+    if (!commentResponse) return Response(res, 404, 'Comment not found');
+    const { comment } = commentResponse.dataValues;
+
+    const likeComment = await commentlikes.create({
+      userid: id,
+      commentid,
+    });
+    const { id: likeid, userid } = likeComment.dataValues;
+
+    const user = await User.findOne({
```

Comment:
Is this information necessary? Do we need to return the `user` object after liking the comment? @Chiazokam

Reply:
It may be needed if we want to display the people that have liked a comment. 
Say, on hover or based on what the team agrees on

Reply:
It is a call that is too costly. We would implement it if needed. For now, kindly don't make that call

Reply:
Alright

Reply:
Alright